anta/input_models/bfd.py

```diff
@@ -31,6 +31,9 @@ class BFDPeer(BaseModel):
     """Multiplier of BFD peer. Required field in the `VerifyBFDPeersIntervals` test."""
     protocols: list[BfdProtocol] | None = None
     """List of protocols to be verified. Required field in the `VerifyBFDPeersRegProtocols` test."""
+    detection_time: int | None = None
+    """Detection time of BFD peer in milliseconds. The period of time without receiving BFD packets
+    after which the session is determined to have failed."""
```

Comment:
```suggestion
    """Detection time of BFD peer in milliseconds. Defines how long to wait without receiving BFD packets before declaring the peer session as down.
    
    Optional field in the `VerifyBFDPeersIntervals` test.
    """
```
Please update the test logic + docstring to accomodate this optional field (to avoid breaking changes).

Reply:
Updated. Thanks!!